pages/content/amp-dev/index.html

```diff
@@ -1,5 +1,7 @@
 ---
 $title: Start
+$titles:
+  title: AMP - a web component framework to easily create user-first web experiences
```

Comment:
Why not replace `start` with the correct title?

Reply:
Not the case but when the homepage would be met by any of the navigation walkers (sidebar, header) they would print this full-blown title.

We could flip the pattern and use `$titles.nav` instead for navigation titles.

Reply:
> We could flip the pattern and use `$titles.nav` instead for navigation titles.

SGTM - that would make it more clear

Reply:
Cool, will ping you when done 🙂